packages/bbui/src/Form/Core/Checkbox.svelte

```diff
@@ -19,7 +20,9 @@
 </script>
 
 <label
-  class="spectrum-Checkbox spectrum-Checkbox--emphasized {sizeClass}"
+  class="spectrum-Checkbox spectrum-Checkbox--emphasized {sizeClass} {indeterminate
+    ? 'is-indeterminate'
+    : ''}"
```

Comment:
Not a blocker, but just letting you know that svelte has a shorthand way of applying classes - you can simply do:
```html
class:is-indeterminate={indeterminate}
```
And you can have multiple class tags, so this can simply be:
```html
...
class="spectrum-Checkbox spectrum-Checkbox--emphasized {sizeClass}"
class:is-indeterminate={indeterminate}
...
```

Reply:
Oh! That's handy :)